### ISSUE TITLE
Missing some Ana-related monsters / relationships

### ISSUE THREAD

**Issue by DreadPirateShawn** [madcowfred/padherder-issues#966]
Ana [3086] and her evo [3087] are present.
Divine Law Goddess Armor, Ana [3088] is missing.
Holy Ceres Armor, Ana [3089] is present and in the evo chain of Ceres [126].
Chaotic Black Dragonbound Armor, Ana [3090] is present, but is not in the evo chain of Black Dragonbound, Typhon [1948].

**Comment by JHunz** [comment]
Yeah, we know.  There's something wrong with all of the ana (and some of the ace) evos, probably left over from when the data was wrong when originally scraped.  We're already looking into it

**Comment by DreadPirateShawn** [comment]
Fair enough. And I just saw #954, didn't have "Ana" in it so my courtesy search missed it. Sorry to pester you on the matter! :-)

**Comment by noip** [comment]
Typhon Ana (3090) not being in Typhon's evo path is still an issue

**Comment by ryuujinzero** [comment]
We know, that's why this issue is still open.

**Comment by damnthefall** [comment]
Just wondering if Divine Law Goddess Armor, Ana [3088] would be added anytime soon?

**Comment by Kodeplay10** [comment]
As an update, Ana Valkyrie evolution is still missing. Her ID cannot be found when trying to add a new monster as well

**Comment by Raijinili** [comment]
ValkAna is the first case of UUUEvo, and PDX's tables were designed to only have two columns for ults.

ValkAna will not be fixed until one of the following happens:
1. PDX fixes it, and PADh's PDX scraper adapts to the fix.
2. PADh hardcodes ValkAna in.
3. PADh stops using PDX data.

Unfortunately, both the PDX devs and the PADh dev aren't really playing the game anymore, as far as I can tell. I'm supposed to make number 3 happen, but I'm not.

**Comment by wmedlar** [comment]
The bug has been fixed and is currently in review.
